plugins/plugin-bash-like/src/pty/server.ts

```diff
@@ -91,9 +91,9 @@ export const disableBashSessions = async (): Promise<ExitHandler> => {
     }
   }
 
-  return async (exitCode: number) => { /* no-op */ }
```

Comment:
maybe change the return type as well?

Reply:
Can you give me more details please?

Reply:
The return type here is ```Promise<ExitHandler>```. If you eliminate ```exitCode``` here, typescript compiler should error out. I'm not sure why it didn't, probably we have some bugs here. Anyways, I suggest changing ```type ExitHandler = (exitCode: number) => Promise<void>``` to ```type ExitHandler = (exitCode?: number) => Promise<void>```  for type consistency.

Reply:
nice catch @myan9 

hmm, i believe we would need to enforce strictNullChecks in order to have typescript catch this. it's pretty clear that we need to work towards that!

Reply:
I think we should keep the focus in the linting, which is the goal of this PR. We can improve things in new ones.

Reply:
Regarding to types, only removing `exitCode` here is wrong. You should change the return type as well. It's just 1 character change. The purpose of this PR is linting, but we don't want to introduce wrong thing, do we? Thanks!